packages/leancode_contracts/pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: leancode_contracts
 repository: https://github.com/leancodepl/contractsgenerator-dart
 description: Dart contracts runtime for a CQRS API.
```

Comment:
Does this package really provide a _runtime_?

Reply:
In a sense, yes. But I understand it might be a confusing phrasing